mcfix_account/models/product.py

```diff
@@ -5,13 +5,6 @@
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
 
-    taxes_id = fields.Many2many(
-        domain="[('type_tax_use', '=', 'sale'),"
-               "('company_id', '=', company_id)]")
```

Comment:
You can put `'|', ('company_id', '=', company_id), ('company_id', '=', False)]` instead, right?